src/components/[guild]/AccessHub/components/AccessedGuildPlatformCard.tsx

```diff
@@ -2,45 +2,53 @@ import PlatformCard from "components/[guild]/RolePlatforms/components/PlatformCa
 import { useRolePlatform } from "components/[guild]/RolePlatforms/components/RolePlatformProvider"
 import useGuild from "components/[guild]/hooks/useGuild"
 import useGuildPermission from "components/[guild]/hooks/useGuildPermission"
-import rewards from "rewards"
 import { cardPropsHooks } from "rewards/cardPropsHooks"
-import rewardComponents from "rewards/components"
+import rewardComponentsConfig from "rewards/components"
 import { PlatformName, PlatformType } from "types"
 import PlatformAccessButton from "./PlatformAccessButton"
 
 const AccessedGuildPlatformCard = () => {
-  const { guildPlatform } = useRolePlatform()
+  const rolePlatform = useRolePlatform()
```

Comment:
Nit: why not keep the {guildPlatform} destruction here? Since we only use that from rolePlatform, felt cleaner to me

Reply:
Oh, good point, I originally passed down `rolePlatform` as a prop to the `PlatformCardWarning` component, but then realised that I could just use the `useRolePlatform` hook & just forgot to revert this change. Will do it before merging the PR, thanks!